src/problems/MetalAdvectionProblem/test_sne.cpp

```diff
@@ -0,0 +1,562 @@
+
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file test_hydro3d_blast.cpp
+/// \brief Defines a test problem for a 3D explosion.
+///
+
+#include <iostream>
+#include <limits>
+#include <math.h>
```

Comment:
You can replace this with `#include <cmath>`. `math.h` is the old C header, whereas `cmath` is the C++ header.